### PR TITLE
User UUID

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: test/testContext.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ let survey = null
  * before executing all tests
  */
 const initTestContext = async () => {
-  user = await UserService.fetchUserByCognitoUsername('demo_user')
+  user = await UserService.fetchUserByUuid('demo_user')
 }
```
Comment (minotogna) [inline test/testContext.js]: Is the uuid demo_user?

Comment (fefrain) [reply test/testContext.js]: I don't think so...! That was the username...

Comment (minotogna) [reply test/testContext.js]: so are the tests working??

Comment (sgiaccio) [reply test/testContext.js]: Fixed, they work now

**Pull request merged**